src/app/site/footer/footer.component.ts

```diff
@@ -15,7 +16,7 @@ export class FooterComponent implements OnInit {
   isMobile = false;
 
 
-  constructor(private getJsonFileService: GetJsonFileService, private responsiveService: ResponsiveService) {
+  constructor(private getJsonFileService: GetJsonFileService, private responsiveService: ResponsiveService, protected router: Router) {
```

Comment:
do we need protected router?

Reply:
it's need for navigate to about-us page(in future if we want to use from goToAnchore method to scroll to correct point, we must use angular router, href can't help us lonely)